Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferSetProxy.h

```diff
@@ -88,7 +88,11 @@ class RemoteImageBufferSetProxy : public RefCounted<RemoteImageBufferSetProxy>,
 
     WebCore::RenderingResourceIdentifier displayListResourceIdentifier() const { return m_displayListIdentifier; }
 
-    std::unique_ptr<ThreadSafeImageBufferSetFlusher> flushFrontBufferAsync();
+    enum class FlushType {
```

Comment:
Can we share the same `enum` with `RemoteLayerBackingStore` ?

Reply:
Yeah, I guess we can. I'll make it a property of the flusher, since `ImageBufferSet` can't use things from `RemoteLayerBackingStore`.